doc/man1/cms.pod

```diff
@@ -553,6 +559,39 @@ The B<-debug_decrypt> option can be used to disable the MMA attack protection
 and return an error if no recipient can be found: this option should be used
 with caution. For a fuller description see L<CMS_decrypt(3)>).
 
+=head1 CAdES Basic Electronic Signature (CAdES-BES)
+
+A CAdES Basic Electronic Signature (CAdES-BES) contains:
+
```

Comment:
This section does not mention RFC 5126 at all. Neither does it mention ETSI EN 319 122-1 V1.1.1.